spec/rails_helper.rb

```diff
@@ -25,7 +25,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-# Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require f }
+Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require f }
```

Comment:
So I read the comment block above, and it seems that this line may impact how fast our specs run.

Reply:
Yeah, it might. I guess the argument is that we'll probably need the login helper spec for pretty much every test